built_value/lib/src/regexp_serializer.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015, Google Inc. Please see the AUTHORS file for details.
+// All rights reserved. Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import 'package:built_collection/built_collection.dart';
+import 'package:built_value/serializer.dart';
+
+/// Runtime type for [RegExp] private implementation.
+final _runtimeType = RegExp('').runtimeType;
+
+class RegExpSerializer implements PrimitiveSerializer<RegExp> {
+  final bool structured = false;
```

Comment:
Not sure what this is for?

Reply:
That's copied from StringSerializer :) removed